### PR TITLE
Properly hide spinner layers until the end of the animation

### PR DESCRIPTION
**Pull request by keanulee**
cc @robdodson

### CODE REVIEW

File: paper-spinner-styles.html [hunk 1]
```diff
@@ -166,7 +166,7 @@
         26% { opacity: 0; }
         89% { opacity: 0; }
         90% { opacity: 0.99; }
-        100% { opacity: 0.99; }
+        to { opacity: 0.99; }
```
Comment (frankiefu) [inline paper-spinner-styles.html]: I thought `to` is the same as `100%`, or are you just trying to be consistent?

Comment (keanulee) [inline paper-spinner-styles.html]: Just consistency - and to save 2 bytes :p

Comment (frankiefu) [inline paper-spinner-styles.html]: Sounds good :-)

**Comment by frankiefu** [comment]
LGTM

**Pull request merged**